solidity/contracts/converter/BancorConverterUpgrader.sol

```diff
@@ -66,7 +68,8 @@ contract BancorConverterUpgrader is IBancorConverterUpgrader, ContractRegistryCl
       * 
       * @param _registry    address of a contract registry contract
     */
-    constructor(IContractRegistry _registry) ContractRegistryClient(_registry) public {
+    constructor(IContractRegistry _registry, IEtherToken _etherToken) ContractRegistryClient(_registry) public {
+        etherToken = _etherToken;

```

Comment:
no need to change the interface, can just hardcode the ether token address here

Reply:
@yudilevi : It would be very hard to properly test this contract without it (I can add an only-owner setter, but I prefer it this way). And this is actually a crucial test in our system (been working on it for about two days now).
My thought was that changing the constructor isn't an API-breaking change. Am I wrong?

Reply:
yeah if it makes testing easier, sure - but how are we gonna test the converter that has the ether token address hardcoded in it?